src/services/logger.ts

```diff
@@ -0,0 +1,22 @@
+import { Logger } from '@aws-lambda-powertools/logger'
+import { LogLevel } from '@aws-lambda-powertools/logger/lib/types/Log'
+import { Context } from 'aws-lambda'
+
+const getLogLevel = (): LogLevel => {
+  if (process.env.LOG_LEVEL) {
+    return process.env.LOG_LEVEL as LogLevel
```

Comment:
Shouldn't really need to declare `as LogLevel` here because you've defined the return type — was it complaining about something?